src/jit/codegencommon.cpp

```diff
@@ -9406,9 +9422,6 @@ void                CodeGen::genFnEpilog(BasicBlock* block)
             compiler->unwindBegEpilog();
             unwindStarted = true;
         }
-        // mov FP into SP
-        inst_RV_RV(INS_mov, REG_SPBASE, REG_FPBASE);
-        compiler->unwindSetFrameReg(REG_FPBASE, 0);
```

Comment:
Can you provide an example of how I should set up unwind code for this case -- we restore sp from fp?